packages/formation-react/src/components/Telephone/Telephone.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const CONTACTS = Object.freeze({
+  '222_VETS': '8772228387', // VA Help Line
+  '4AID_VET': '8774243838', // National Call Center for Homeless Veterans
+  911: '911',
+  CAREGIVER: '8552603274', // VA National Caregiver Support Line
+  CRISIS_LINE: '8002738255', // Veterans Crisis hotline
+  CRISIS_TTY: '8007994889', // Veterans Crisis hotline TTY
+  DS_LOGON: '8005389552', // Defense Manpower Data Center
+  DS_LOGON_TTY: '8663632883', // Defense Manpower Data Center TTY
+  GI_BILL: '8884424551', // Education Call Center (1-888-GI-BILL-1)
+  GO_DIRECT: '8003331795', // Go Direct/Direct Express (Treasury)
+  HELP_DESK: '8555747286', // VA Help desk
+  HELP_TTY: '8008778339', // VA Help Desk TTY
+  MY_HEALTHEVET: '8773270022', // My HealtheVet help desk
+  NCA: '8005351117', // National Cemetery Scheduling Office
+  TESC: '8882242950', // U.S. Treasury Electronic Payment Solution Center
+  VA_311: '8446982311', // VA Help desk (VA311)
+  VA_BENEFITS: '8008271000', // Veterans Benefits Assistance
+});
+
+export const PATTERNS = {
+  911: '###', // needed to match 911 CONTACT
+  DEFAULT: '###-###-####',
+  WRAP_AREACODE: '(###) ###-####',
+};
+
+// Strip out leading "1" and any non-digits
+const parseNumber = number =>
+  number
+    .replace(/^1/, '') // strip leading "1" from telephone number
+    .replace(/[^\d]/g, '');
+
+// Create link text from pattern
+const formatTelText = (num, pattern) => {
+  let i = 0;
+  return pattern.replace(/#/g, () => num[i++] || '');
+};
+
+const formatBlock = number => {
+  const len = number.length - 1;
+  // return rounded number as a grouped value, e.g. `800` or `1000`
+  // but split non-round numbers, e.g. `827` => `8 2 7`
```

Comment:
```suggestion
  // group when divisible by ten, otherwise split with space
  // e.g. `800`, `1000`, `8 2 7` 
```

Reply:
I added JSDocs to each function, and got a little more verbose:

> Group number parts that are divisible by 1000 or 100 (chose the greater one, if possible) and spread out other numbers; we want the screenreader to sound out the numbers as we typically say them, e.g. "eight hundred five five five one thousand" (800-555-1000)